packages/composables/src/use-collection.ts

```diff
@@ -1,17 +1,17 @@
 import type { AppCollection, Field } from '@directus/types';
-import type { ComputedRef, Ref } from 'vue';
+import type { Ref } from 'vue';
 import { computed, ref } from 'vue';
 import { useStores } from './use-system.js';
 
 export type UsableCollection = {
-	info: ComputedRef<AppCollection | null>;
-	fields: ComputedRef<Field[]>;
-	defaults: ComputedRef<Record<string, any>>;
-	primaryKeyField: ComputedRef<Field | null>;
-	userCreatedField: ComputedRef<Field | null>;
-	sortField: ComputedRef<string | null>;
-	isSingleton: ComputedRef<boolean>;
-	accountabilityScope: ComputedRef<'all' | 'activity' | null>;
+	info: Ref<AppCollection | null>;
+	fields: Ref<Field[]>;
+	defaults: Ref<Record<string, any>>;
+	primaryKeyField: Ref<Field | null>;
+	userCreatedField: Ref<Field | null>;
+	sortField: Ref<string | null>;
+	isSingleton: Ref<boolean>;
+	accountabilityScope: Ref<'all' | 'activity' | null>;
```

Comment:
Considering `useCollection` actually returns `ComputedRef`s should types not reflect this? Having these types as Ref may give the false impression that these values are writable instead of computed

Reply:
Good point, thanks! Fixed now at other places too: 9c2a838